src/opencensus.app.src

```diff
@@ -0,0 +1,16 @@
+{application, opencensus,
+ [{description, "Erlang stats collection and distributed tracing framework"},
+  {vsn, "0.1.0"},
```

Comment:
perhaps use git versioning, I found it quite helpful to ensure tagging happens and links to packages correctly.

Reply:
I can never decide anymore. On the one hand I like that it appends a `+build.refxxx` to the version when not on a tag, on the other I don't like how it requires shelling out for every app that uses `git` version when it is built and it breaks for anyone who downloads the `.zip` from github since it doesn't have the git information.